KanvasExample/KanvasExampleTests/Editor/EditorControllerTests.swift

```diff
@@ -135,8 +135,8 @@ final class EditorControllerTests: FBSnapshotTestCase {
         let cameraSettings = settings ?? getCameraSettings()
         let handler = assetsHandler ?? AssetsHandlerStub()
         let analytics = analyticsProvider ?? KanvasAnalyticsStub()
-        let viewController = EditorViewController(settings: cameraSettings, segments: segments, assetsHandler: handler, exporterClass: MediaExporterStub.self, gifEncoderClass: GIFEncoderStub.self, cameraMode: cameraMode, stickerProvider: StickerProviderStub(), analyticsProvider: analytics, quickBlogSelectorCoordinator: nil, tagCollection: nil)
-        viewController.delegate = delegate ?? newDelegateStub()
+        let delegate = delegate ?? newDelegateStub()
+        let viewController = EditorViewController(delegate: delegate, settings: cameraSettings, segments: segments, assetsHandler: handler, exporterClass: MediaExporterStub.self, gifEncoderClass: GIFEncoderStub.self, cameraMode: cameraMode, stickerProvider: StickerProviderStub(), analyticsProvider: analytics, quickBlogSelectorCoordinator: nil, tagCollection: nil)
```

Comment:
It looks like this is breaking the unit tests right now since we're not going to pass an `EditorControllerDelegate` in the initializer anymore.